ckanext/harvest/templates/index.html

```diff
@@ -27,78 +21,95 @@
+
+      <h1>Harvesting sources</h1>
+
+      <div class="harvest-content boxed">
+        {% if can_edit and c.status %}
+        <div class="status">
+          <h3>Status:</h3>
+          {{h.literal(c.status)}}
+        </div>
+        {% endif %}
+
+        {% if org_ids %}
+        <div>
+            {% if can_edit %}
+            <div id="show-inactive-sources-content">
+              <input type="checkbox" id="show-inactive-sources" />
+              <label for="show-inactive-sources"> Show inactive sources</label>
+            </div>
+            {% endif %}
+            <h3>My organizations</h3>
+            {{harvest_source_table(h.allowed_sources(org_ids), can_edit)}}
+        </div>
+        {% endif %}
+        {% if c.sources %}
+            {% if can_edit %}
+            <div id="show-inactive-sources-content">
+              <input type="checkbox" id="show-inactive-sources" />
+              <label for="show-inactive-sources"> Show inactive sources</label>
+            </div>
+            {% endif %}
+            <h3>All organizations</h3>
+            {{harvest_source_table(c.sources, can_edit)}}
+        {% else %}
+              <div id="no-harvest-sources">No harvest sources defined yet.</div>
+        {% endif %}
+  </div>
+</div>
 
-  </py:def>
-
-    <div py:match="content" py:with="can_edit=h.check_access('harvest_source_create'); org_ids=[o.id for o in h.organizations_available('create_dataset')] if c.userobj else []">
-
-        <a py:if="can_edit" class="btn btn-primary pull-right" id="xnew-harvest-source" href="harvest/new">
-            <i class="icon-plus"></i>
-            Add a harvesting source
-        </a>
-
-        <h1>Harvesting sources</h1>
-
-        <div class="harvest-content boxed">
-
-
-          <div py:if="can_edit and c.status" class="status">
-            <h3>Status:</h3>
-            ${h.literal(c.status)}
-          </div>
-
-          <div py:if="org_ids">
-              <div py:if="can_edit" id="show-inactive-sources-content">
-                <input type="checkbox" id="show-inactive-sources" />
-                <label for="show-inactive-sources"> Show inactive sources</label>
-              </div>
-              <h3>My organizations</h3>
-              ${harvest_source_table((s for s in c.sources if s.publisher_id in org_ids), can_edit)}
-          </div>
-          <py:choose>
-            <py:when test="c.sources">
-              <div py:if="can_edit" id="show-inactive-sources-content">
-                <input type="checkbox" id="show-inactive-sources" />
-                <label for="show-inactive-sources"> Show inactive sources</label>
-              </div>
 
-              <h3>All organizations</h3>
-              ${harvest_source_table(c.sources, can_edit)}
+{% endblock%}
 
-            </py:when>
-            <py:otherwise>
-                <div id="no-harvest-sources">No harvest sources defined yet.</div>
-            </py:otherwise>
-          </py:choose>
+{% block optional_footer %}
```

Comment:
BTW I can't find optional_footer in the jinja templates - I think this should just be moved to the bottom of the block above it.